meson.build

```diff
@@ -513,7 +513,7 @@ if packager_version != ''
 endif
 rz_version_h = configure_file(
   input: 'librz/include/rz_version.h.in',
-  output: 'rz_version.h',
+  output: 'rz_build_version.h',
```

Comment:
The build should work even if there are 2 different `rz_version.h` files because they are in different folders (one is in `include/` and one is in `include/rz_util/`), but I'm fine with renaming one of them.